apps/api/services/zpe/result_store.py

```
@@ -0,0 +1,133 @@
+        else:
+            raise ValueError("kind は summary / freqs のいずれかです。")
+        raw = cast(Optional[bytes], self.redis.get(key))
+        if raw is None:
+            raise KeyError("file not found")
+        return _decode(raw) or ""
+
+
+def get_result_store() -> ResultStore:
+    settings = get_zpe_settings()
+    if settings.result_store != "redis":
+        raise ValueError("result_store は redis のみ対応しています。")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)